test/hotspot/jtreg/vmTestbase/nsk/share/locks/DeadlockMaker.java

```diff
@@ -27,8 +27,8 @@
 import nsk.share.Wicket;
 
 /*
- * Class used to create deadlocked threads. It is possible create 2 or more deadlocked thread, also
- * is is possible to specify resource of which type should lock each deadlocked thread
+ * Class used to create deadlocked threads. It is possible to create 2 or more deadlocked thread, also
+ * it is possible to specify resource of which type should lock each deadlocked thread
```

Comment:
Even with the corrections this comment still makes little sense, but lets not get bogged down trying to improve these old tests. If you feel like it s/thread,/threads,/ and add a full-stop at the end of the comment.

Reply:
Done. I've updated the PR to include this suggested text.